NAMESPACE

```diff
@@ -61,3 +61,6 @@ export(semi.sup)
 export(sort_samples_by_acquisition_number)
 export(two.step.hybrid)
 export(unsupervised)
+importFrom(dplyr,arrange)
+importFrom(dplyr,filter)
+importFrom(magittr,"%>%")
```

Comment:
instead of manually adding them in the namespace file just add them using roxygen2 at the top of the script which is using the library

Reply:
I did not add it manually. I used `devtools::document()` for that (the same as `roxygen2::roxygenise()`).

Reply:
great! diff were not loading for the prof.to.feature so I couldn't see. Another note all the other files have the imports at the top of the file, to keep it consistent we can move these to the top as well

Reply:
There was a merge conflict so the changes to the NAMESPACE didn't get pushed and I didn't realise this. It should look fine after b68615b.